test/org/javarosa/model/xform/SMSSerializingVisitorTest.java

```diff
@@ -0,0 +1,161 @@
+package org.javarosa.model.xform;
+
+import org.javarosa.core.PathConst;
+import org.javarosa.core.model.instance.FormInstance;
+import org.javarosa.core.services.transport.payload.ByteArrayPayload;
+import org.javarosa.core.test.FormParseInit;
+import org.javarosa.form.api.FormEntryController;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.javarosa.test.utils.Utils.convertToString;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Performs a comprehensive suite of tests on the results of a form
+ * being serialized to the SMS format.
+ */
+public class SMSSerializingVisitorTest {
```

Comment:
I realize that SMSSerializingVisitor is an existing class, so I wouldn’t ask you to correct its name to SmsSerializingVisitor for this PR, but SMSSerializingVisitorTest is a new class, so how about calling it SmsSerializingVisitorTest? I think following the naming convention violation is more important than temporary inconsistency between the test class name and the class being tested.